conjureup/app.py

```diff
@@ -63,6 +66,9 @@ def parse_options(argv):
                         help='Location of conjure-up managed spells directory',
                         default=os.path.expanduser(
                             "~/.cache/conjure-up-spells"))
+    parser.add_argument('--conf-file', dest='conf_file',
+                        help='Path to configuration file', type=pathlib.Path,
+                        default=pathlib.Path("~/.config/conjure-up.conf"))
```

Comment:
It should be the same as the `.cache` folder, in that due to `--classic` it should be outside the snap.